client/app/components/product-card/product-name.jsx

```diff
@@ -1,20 +1,30 @@
 const React = require('react');
+const PropTypes = require('prop-types');
 const style = require('styled-components').default;
-const Container = require('../lib/container.jsx');
-const Heading = require('../lib/heading.jsx');
 
-const NameContainer = style(Container)`
+const Container = style.div`
```

Comment:
NameContainer seemed like a better descriptive variable here!

Reply:
I thought that too. I am still contemplating what to do about that. This short version is less verbose.